packages/xo-server/src/xapi/index.mjs

```diff
@@ -1052,26 +1052,30 @@ export default class Xapi extends XapiBase {
       )
     } catch (error) {
       const { code } = error
-      if (code === 'VDI_CBT_ENABLED') {
+      if (code === 'VDI_CBT_ENABLED' && !_failOnCbtError) {
+        log.debug(`${vdi.name_label} has CBT enabled`)
         // 20240629 we need to disable CBT on all disks of the VM since the xapi
         // checks all disk of a VM even to migrate only one disk
-        if (vdi.VBDs.length === 0) {
+        const vbds = vdi.$VBDs.filter(({ $VM }) => $VM.is_control_domain === false)
```

Comment:
I don't understand this change in this PR.

At least add a comment, otherwise we will not understand why we did this in the future.